openmdao/docs/feature_reference/building_blocks/solvers/linear/scipy_iter_solver.rst

```diff
@@ -1,10 +1,10 @@
 .. _scipyiterativesolver:
 
 ********************
-ScipyIterativeSolver
+ScipyGMRES
 ********************
 
-The ScipyIterativeSolver is an iterative linear solver that wraps the methods found in `scipy.sparse.linalg`.
+The ScipyGMRES is an iterative linear solver that wraps the methods found in `scipy.sparse.linalg`.
```

Comment:
I think "The" should go away...  (or the word "solver" added)

Reply:
fixed